### ISSUE TITLE
Improve "unexpected" failure: include following/extra messages

### ISSUE THREAD

**Issue by blueyed** [davidfritzsche/pytest-mypy-testing#4]
It would be useful if following messages would also be displayed in case of
mismatches, e.g. with these message (in `diff_message_sequences`):

```
(Pdb++) pp actual_messages
[Message(filename='…/test_typing.py', lineno=1, colno=1, severity=<Severity.ERROR: 3>, message="No library stub file for module 'foo'"),
 Message(filename='…/test_typing.py', lineno=1, colno=1, severity=<Severity.NOTE: 1>, message='(Stub files are from https://github.com/python/typeshed)'),
 Message(filename='…/test_typing.py', lineno=7, colno=11, severity=<Severity.ERROR: 3>, message='Incompatible types in assignment (expression has type "int", variable has type "str")')]
(Pdb++) expected_messages
[Message(filename='…/test_typing.py', lineno=7, colno=None, severity=<Severity.ERROR: 3>, message='Incompatible types in assignment (expression has type "int", variable has type "str")')]
```

You only see:
```
============ FAILURES ============
___ [mypy]mypy_use_reveal_type ___
…/test_typing.py:1: note (unexpected): (Stub files are from https://github.com/python/typeshed)
```

So the actual message is there already, but only after initial error/notes.

**Comment by davidfritzsche** [comment]
@blueyed  Do I understand this correctly, that the message "No library stub file for module 'foo'" is not shown? If yes, do you have a small example that exposes this issue?

**Comment by blueyed** [comment]
@davidfritzsche 
Yeah, looks like it.  The note only happens with the error after all (but I might have mixed the example output up / changed it while in pdb).

The example is:
```python
import pytest


@pytest.mark.mypy_testing
def mypy_use_reveal_type() -> None:
    foo = "abc"
```

I've just tried it again and the output is the same.

From mypy directly:
```
test_typing.py:1: error: No library stub file for module 'pytest'
test_typing.py:1: note: (Stub files are from https://github.com/python/typeshed)
test_typing.py:7: error: Incompatible types in assignment (expression has type "int", variable has type "str")
```

So there's another issue there even (besides that it would be helpful to display all three (including the extra) lines).

(I've only played a bit with it for now, not much experience with it yet)

**Comment by davidfritzsche** [comment]
@blueyed Thank you for testing out the plugin. There was indeed the generic problem that in the diff processing only one message per source-code line would be considered (the last one in the diff winning). This should work better now with v0.0.4, see #5

**Comment by davidfritzsche** [comment]
I'm closing this issue for now.

**Comment by blueyed** [comment]
Thanks for fixing it already!
Did not have time / need to use it since then, but will report any new findings.. :)
